<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginMotorDialog</class>
 <widget class="QDialog" name="LoginMotorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1741</width>
    <height>1304</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>HP Simplified</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QPushButton" name="buttonLogin">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>770</y>
     <width>531</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kirjaudu sisään</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPW">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>150</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelErrorText">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>80</y>
     <width>521</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>errorText</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancelLogin">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>270</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Keskeytä</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonNumpad2">
   <property name="geometry">
    <rect>
     <x>776</x>
     <y>261</y>
     <width>89</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonNumpad1">
   <property name="geometry">
    <rect>
     <x>681</x>
     <y>261</y>
     <width>89</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonNumpad3">
   <property name="geometry">
    <rect>
     <x>871</x>
     <y>261</y>
     <width>89</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonNumpad7">
   <property name="geometry">
    <rect>
     <x>681</x>
     <y>509</y>
     <width>89</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonNumpad8">
   <property name="geometry">
    <rect>
     <x>776</x>
     <y>509</y>
     <width>89</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonNumpad9">
   <property name="geometry">
    <rect>
     <x>871</x>
     <y>509</y>
     <width>89</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonNumpad5">
   <property name="geometry">
    <rect>
     <x>776</x>
     <y>385</y>
     <width>89</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonNumpad4">
   <property name="geometry">
    <rect>
     <x>681</x>
     <y>385</y>
     <width>89</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonNumpad6">
   <property name="geometry">
    <rect>
     <x>871</x>
     <y>385</y>
     <width>89</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonNumpadClear">
   <property name="geometry">
    <rect>
     <x>871</x>
     <y>633</y>
     <width>89</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pyyhi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonNumpad0">
   <property name="geometry">
    <rect>
     <x>776</x>
     <y>633</y>
     <width>89</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonNumpadNoUse1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>681</x>
     <y>633</y>
     <width>89</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
